recipe/meta.yaml

```
@@ -9,13 +9,15 @@ source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: a3672c474dd094e2fefd364757d8fb6aa016af6e2ffee356e2a9a78c88f58690
   patches:
-    # Patch `pyproject.toml`'s requirement on `cupy-cuda12x` to `cupy` to fix `pip check`.
+    # Patch `pyproject.toml`'s requirement on `cupy-cuda12x` to `cupy` to fix `pip check`
     - cupy.patch
 
 build:
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
-  skip: True  # [win or osx]
+  number: 2
+  # linux-ppc64le is not supported by CuPy on Python 3.13
+  # https://github.com/conda-forge/cupy-feedstock/issues/299
+  skip: True  # [ppc64le and py>312]
```

```suggestion
  skip: True  # [win or osx or (ppc64le and py>312)]
```

Ah this used to be

```
skip: True # [win or osx]
```

and we still need those too!

---

Ah, I did not re-render, that's why the CI wasn't failing. Thanks!